curation-migration-datasync/types.ts

```diff
@@ -12,11 +12,28 @@ export type ScheduledItemPayload = {
   isSyndicated: boolean;
   createdAt: number;
   createdBy: string;
+  updatedBy?: string;
   updatedAt: number;
   scheduledSurfaceGuid: string;
   scheduledDate: string;
 };
 
+export type ApprovedItemPayload = {
+  eventType: string;
+  approvedItemExternalId: string;
+  url: string;
+  title: string | null;
+  excerpt: string | null;
+  language: string | null;
+  publisher: string | null;
+  imageUrl: string | null;
+  topic: string | null;
+  createdAt: number | null;
+  createdBy: string | null;
+  updatedAt: number | null;
```

Comment:
These fields aren't nullable -- they are optional (might be undefined)

Reply:
modified them as optional field, so if they are set to null or undefined, we won't set them